starter-code/scripts/articleView.js

```diff
@@ -18,29 +18,45 @@ articleView.populateFilters = function() {
 
 articleView.handleAuthorFilter = function() {
   $('#author-filter').on('change', function() {
-  if ($(this).val()) {
-      /* TODO: If the slect box changes to an option that has a value, we should:
+    if ($(this).val()) {
+      $('article').hide();
```

Comment:
It is generally a good practice to be as specific as possible with your selectors. You might have articles in your HTML that aren't related to your blog articles, and you probably wouldn't want to hide them. Instead, you could write it this way to ensure that you're getting the expected behavior:
```
$('#articles article').hide();
```